scripts/e2e-test.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env sh
+
+set -e
+
+SCRIPTDIR=$(dirname "$0")
+REGISTRY=$1
+TESTS="install"
+
+# TODO: Add proper argument parser
+if [ -z "$REGISTRY" ]; then
+  echo "Missing parameter registry"
+  exit 1
+fi
+
+test_failed=
+export e2e_docker_registry="$REGISTRY"
+export e2e_pool_device=/dev/nvme1n1
+
+for dir in $TESTS; do
+  cd $SCRIPTDIR/../mayastor-test/e2e/$dir
```

Comment:
Just a nitpick of shellcheck (not mine :smile:):

```suggestion
  cd "$SCRIPTDIR/../mayastor-test/e2e/$dir"
```

Reply:
fixed. thanks!